web/src/components/DisputePreview/DisputeContext.tsx

```diff
@@ -45,11 +42,15 @@ const AnswersContainer = styled.div`
   flex-direction: column;
 `;
 
+const AnswersHeader = styled.h3`
+  margin: 0;
+`;
+
 const Answer = styled.div`
   margin: 0px;
   display: flex;
   flex-wrap: wrap;
-  gap: ${responsiveSize(2, 8)};
+  gap: 6px;
```

Comment:
Here I would maintain the `responsiveSize` and if we want to decrease the maximum gap, let's set it to:
```
gap: ${responsiveSize(2, 6)};
```
This way we are reducing the maximum gap (from 8px to 6px) while maintaining the possibility to go lower on mobile devices.